### ISSUE TITLE
asset name length clarification

### ISSUE THREAD

**Issue by zoedberg** [RGB-WG/rgb#8]
While trying to issue an asset I've received an error because `name` didn't meet the requirement of being at least 5 chars long. I'm wondering why this limitation is there, before (in v0.9) it was limited from 1 up to 256 chars. @dr-orlovsky could you please clarify this change?

**Comment by dr-orlovsky** [comment]
Yes, to prevent creation of meaningless asset names. Name should be longer than a ticker.

Now, with interfaces, we have more freedom and there could be different token standards. I am opened to change this one, since the current interface is anyway not a complete implementation of RGB20

**Comment by fedsten** [comment]
There are many cases where asset names are shorter than 5 chars, for example among the top shitcoins by marketcap you can find Tron, BNB, DAI, XRP, EOS and many others. Being the name just a metadata of the asset I do not see any potential issue in leaving more freedom to the issuer.

**Comment by dr-orlovsky** [comment]
Agree. I actually recall now why I "added" this limit: since I didn't made that consciously but just used an existing data type `Ident` (which I use for Strict Type data names).

I think I have to update RGB20 interface to match the standard. The reality is that I was in rush with releasing v0.10 and since interfaces now are flexible and can be updated independently from the lib, I did "RGB20" which is just a minimal viable thing composed of existing components - and not a real RGB20 we need: it has no secondary issuance, wrong name length requirements, etc.

I will work on making a proper RGB20 matching the standard we have (which we developed over many years): https://github.com/LNP-BP/LNPBPs/blob/master/lnpbp-0020.md
